milestones/models.py

```diff
@@ -32,7 +34,7 @@ class Meta:
         unique_together = (("namespace", "name"),)
 
     def __unicode__(self):
-        return unicode(self.namespace)
+        return str(self.namespace)
```

Comment:
This returns the wrong type on Python 2; maybe use the [python_2_unicode_compatible](https://docs.djangoproject.com/en/1.11/topics/python3/#str-and-unicode-methods) decorator instead?